evap/staff/tests/test_views.py

```diff
@@ -7,8 +8,8 @@
 from model_mommy import mommy
 import xlrd
 
-from evap.evaluation.models import Semester, UserProfile, Course, CourseType, \
-                                   TextAnswer, Contribution
+from evap.evaluation.models import Semester, UserProfile, Course, CourseType, TextAnswer, Contribution, Questionnaire,\
+    Question
```

Comment:
yeah, i tried to like the parentheses but, well, for imports they do look worse than the backslash in my opinion.

you can either revert to the previous style or apply your style to the project. while you're at it, you could also make the import order PEP8-compliant, maybe PyCharm can do this automatically as well ;) that would be a change that actually helps...